examples/client/Locomotion/src/context/user/index.tsx

```diff
@@ -173,13 +173,27 @@ const UserContextProvider = ({ children }: { children: any }) => {
     return vertResponse.status === 'OK';
   };
 
+  const getAllowedDemandSourceIds = () => {
+    if (!Config.ALLOWED_DEMAND_SOURCE_IDS) {
+      return [];
+    }
+    const allowedDemandSourceIds = JSON.parse(Config.ALLOWED_DEMAND_SOURCE_IDS);
```

Comment:
add try catch to the JSON parse + mixpanel event. 
As it ui only logic , it will help us to debug if the Config.ALLOWED is invalid json or undefined